### PR TITLE
Okazaki - Other Clinical Attribute - P1

### PR DESCRIPTION
**Pull request by wwysoc2**

### CODE REVIEW

**Review by clsackett — APPROVED** [review]

**Review by clsackett — APPROVED** [review]
Good catch, LGTM

File: src/gdcdictionary/schemas/other_clinical_attribute.yaml [hunk 1]
```diff
@@ -1127,19 +1296,36 @@ properties:
...
       - Post Secondary Therapy
       - Postoperative
       - Preoperative
+      - Prior to Adjuvant Therapy
+      - Prior to Chemotherapy
       - Prior to Diagnosis
+      - Prior to Procurement
+      - Prior to Study Enrollment
+      - Prior to Study Registration
       - Prior to Treatment
```
Comment (wteouchicago) [inline src/gdcdictionary/schemas/other_clinical_attribute.yaml:1324]: note that the ticket listed this as an enum to add - maybe we need to edit the ticket?

File: src/gdcdictionary/schemas/other_clinical_attribute.yaml [hunk 2]
```diff
@@ -174,64 +201,99 @@ properties:
...
         - Gonadal Dysfunction
         - Gorlin Syndrome
         - Gout
         - Graves' Disease
         - Hashimoto's Thyroiditis
+        - Hay Fever
         - Headache
         - Heart Disease
         - Helicobacter pylori Infection
+        - Helicobacter pylori-Associated Gastritis
```
Comment (wteouchicago) [inline src/gdcdictionary/schemas/other_clinical_attribute.yaml:268]: the capitalization seems a bit weird to me here but I've no idea what's right.

**Comment by wteouchicago** [comment]
I didn't spot the deprecation of `undescended_testis_corrected_age` in the PR per the jira

File: src/gdcdictionary/schemas/other_clinical_attribute.yaml (hunk 1 above)
Comment (wwysoc2) [reply src/gdcdictionary/schemas/other_clinical_attribute.yaml:1324]: Removed from ticket,  @bullenc could you remove this also from the spreadsheet?  I removed the duplicate from this PR.

Comment (wwysoc2) [reply src/gdcdictionary/schemas/other_clinical_attribute.yaml:1324]: Actually, nvm, you can just pull the dictionary into your spreadsheet when it's all merged to dev.

File: src/gdcdictionary/schemas/other_clinical_attribute.yaml (hunk 2 above)
Comment (wwysoc2) [reply src/gdcdictionary/schemas/other_clinical_attribute.yaml:268]: This is a species name, so this is odd-looking but correct.

**Comment by wwysoc2** [comment]
@wteouchicago undescended_testis_corrected_age has been added to the deprecated list.

**Review by wteouchicago — APPROVED** [review]

**Review by stilesj-uchicago — APPROVED** [review]
DICT-519 looks correct to me

**Pull request merged**